geode-assembly/src/acceptanceTest/resources/ManagementClientCreateRegion.java

```diff
@@ -31,14 +31,15 @@ public static void main(String[] args) throws Exception {
     if (useSsl) {
       // The default SSLContext will pull in all necessary javax.net.ssl properties
       cms =
-          ClusterManagementServiceBuilder.buildWithHostAddress()
-              .setHostAddress("localhost", httpPort)
-              .setSslContext(SSLContext.getDefault()).build();
+          new ClusterManagementServiceBuilder().setConnectionConfig(
+              new BasicClusterManagementServiceConnectionConfig("localhost", port)
+                  .setSslContext(SSLContext.getDefault()))
+              .buildWithHostAddress();
```

Comment:
shouldn't this be just "build()"?

Reply:
You're right, that file had several errors that I didn't catch because it wasn't being compiled. I fixed this and the other problems.